pkg/brew/bfs.rb

```diff
@@ -0,0 +1,18 @@
+class Bfs < Formula
+  version '1.0.1'
```

Comment:
Perhaps this betrays my unfamiliarity with Homebrew, but why should `master` refer to a released version?

Also, if I tag 1.0.2, it's going to contain the current version of this file, which mentions 1.0.1, which seems confusing.  Perhaps the Homebrew stuff should be excluded from the release archives via `.gitattributes`?